<?xml version="1.0" encoding="UTF-8"?>
<issues format="5" by="lint 4.2.0-beta06" client="gradle" variant="debug" version="4.2.0-beta06">

    <issue
        id="BanTargetApiAnnotation"
        message="Uses @TargetApi annotation"
        errorLine1="@TargetApi(Build.VERSION_CODES.N)"
        errorLine2="~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~">
        <location
            file="src/main/java/androidx/wear/complications/ComplicationHelperActivity.java"
            line="59"
            column="1"/>
    </issue>

    <issue
        id="ClassVerificationFailure"
        message="This call references a method added in API level 29; however, the containing class androidx.wear.utility.AsyncTraceEvent is reachable from earlier API levels and will fail run-time class verification."
        errorLine1="            Trace.beginAsyncSection(traceName, traceId)"
        errorLine2="                  ~~~~~~~~~~~~~~~~~">
        <location
            file="src/main/java/androidx/wear/utility/TraceEvent.kt"
            line="64"
            column="19"/>
    </issue>

    <issue
        id="ClassVerificationFailure"
        message="This call references a method added in API level 29; however, the containing class androidx.wear.utility.AsyncTraceEvent is reachable from earlier API levels and will fail run-time class verification."
        errorLine1="            Trace.endAsyncSection(traceName, traceId)"
        errorLine2="                  ~~~~~~~~~~~~~~~">
        <location
            file="src/main/java/androidx/wear/utility/TraceEvent.kt"
            line="70"
            column="19"/>
    </issue>

</issues>
